notebooks/tools/fortran.ipynb

```diff
@@ -27,7 +66,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.6"
+   "version": "3.9.12"
```

Comment:
It might be nothing important but I was a bit puzzled with the version of python was "downgraded". I think we can use your version for now but maybe ask Brian about that.